README.md

```diff
@@ -10,6 +10,7 @@ Docker image for hugo static page generator (https://gohugo.io)
 * `HUGO_DESTINATION` (Path where hugo will render the site. By default `/output`)
 * `HUGO_REFRESH_TIME` (in seconds, only applies if not watching, if not set, the container will build once and exit)
 * `HUGO_BASEURL`
+* `HUGO_AppendPort` (set to false to prevent the default port to be appended to links)
```

Comment:
I think it would make sense to call it `HUGO_APPEND_PORT` for consistency.

Reply:
updated in subsequent commit to HUGO_APPEND_PORT